lib/src/assets/img/logo-nexus-generic.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Group 7 Copy</title>
+    <g id="Navigation" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Nexus-Headers" transform="translate(-81.000000, -579.000000)">
+            <g id="Group-7-Copy" transform="translate(81.000000, 579.000000)">
+                <rect id="Rectangle" fill="#949494" x="0" y="0" width="72" height="72"></rect>
+                <g id="Group-3" transform="translate(8.000000, 9.000000)" fill="#FFFFFF">
```

Comment:
This `fill` can be moved up to the `<svg>` and this `transform` can be merged with the one below, removing the need for this `<g>`.

Reply:
> This `fill` can be moved up to the `<svg>` and this `transform` can be merged with the one below, removing the need for this `<g>`.

SVG extensively cleaned up here: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/e6ac7a8ea2c55cc21584f554f7ae76b798036799